app/objects/beatmap.py

```diff
@@ -57,33 +57,49 @@ async def api_get_beatmaps(**params: Any) -> Optional[list[dict[str, Any]]]:
     return None
 
 
+KNOWN_BAD_FILES_MD5 = ["d41d8cd98f00b204e9800998ecf8427e"]  # empty file
```

Comment:
absolutely no way i am allowing this through to the codebase. no magic values please.

Reply:
this is a well known md5 for empty files, just like 200, 500 below the code. You can google it.

Reply:
If we don't use magic values how can we tell a file is bad?

Reply:
if you wish to check against an empty file's md5:

- the current example has very poor readability (readability should be independent from comments). Instead, I suggest calculating the md5 for an empty string when the server is started
- store it in something like `EMPTY_FILE_MD5`. why do we need a list of bad hashes? what purpose would this have? feels like this is a bit of scope creep

Reply:
if you need an example to think of how this is done you may check regexes which follow the same pattern using `re.compile`

Reply:
Also how would checking if the file is empty help? If you put a whitespace into it it would go through again

Reply:
- I originally write
```python
KNOWN_BAD_FILES_MD5 = [
    "d41d8cd98f00b204e9800998ecf8427e" # empty file 
]
```
formatter made it like that.


- An empty hashes is for future proof to add more known bad files.

Reply:
yeah my question was WHAT other files could be bad? if you intend to use this for marking random beatmaps as having "bad" files this does not belong in the codebase. if this is intended to be used for multiple "magic" values then this is also going in the wrong direction

Reply:
> Also how would checking if the file is empty help? If you put a whitespace into it it would go through again

Thus I put a list there. Now the api endpoint returns an empty file, and whatever it returns in the future i am planning to add it to the list.

Reply:
https://github.com/osuAkatsuki/bancho.py/pull/428/files#r1167815566 check this